packages/template-chakra-storefront/src/components/list-menu/list-menu-popover.jsx

```diff
@@ -7,52 +7,40 @@
 
 import React, {useState} from 'react'
 import PropTypes from 'prop-types'
-import {Box, Popover, useSlotRecipe} from '@chakra-ui/react'
+import {Popover, useSlotRecipe} from '@chakra-ui/react'
 
 import {ListMenuContent} from '../../components/list-menu/list-menu-content'
 import {ListMenuTrigger} from '../../components/list-menu/list-menu-trigger'
 
 const ListMenuPopover = ({contentComponent, item, name, itemsKey, maxColumns}) => {
     const [open, setOpen] = useState(false)
-    const onOpen = () => setOpen(true)
-    const onClose = () => setOpen(false)
+    const handleOpenChange = (details) => {
+        setOpen(details.open)
+    }
+
     const ContentComponent = contentComponent || ListMenuContent
     const recipe = useSlotRecipe({key: 'listMenu'})
     const styles = recipe()
 
     return (
-        <Box onMouseLeave={onClose}>
-            <Popover.Root
-                open={open}
-                positioning={{placement: 'bottom-start'}}
-                lazyMounted
-                unstyled
-            >
-                <Popover.Trigger asChild>
-                    <Box onMouseEnter={onOpen}>
-                        <ListMenuTrigger
-                            item={item}
-                            name={name}
-                            isOpen={open}
-                            onOpen={onOpen}
-                            onClose={onClose}
-                        />
-                    </Box>
-                </Popover.Trigger>
-                <Popover.Positioner>
-                    <Popover.Content data-testid="popover-menu" css={styles.popoverContent}>
-                        <Popover.Body css={styles.popoverBody}>
-                            <ContentComponent
-                                item={item}
-                                itemsKey={itemsKey}
-                                onClose={onClose}
-                                maxColumns={maxColumns}
-                            />
-                        </Popover.Body>
-                    </Popover.Content>
-                </Popover.Positioner>
-            </Popover.Root>
-        </Box>
+        <Popover.Root
+            positioning={{placement: 'bottom-start'}}
+            lazyMount
```

Comment:
This is the correct spelling of the prop.. not `lazyMounted`. There's a typo in the Chakra docs.